cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,11 +20,16 @@
    ========================================================================== #}
 
 {% macro render(items, active_item_id, additional_classes) %}
-
-<nav class="nav-secondary expandable {{ additional_classes if additional_classes else '' }}"
+{% from 'molecules/nav-link.html' import nav_link as nav_link %}
+<nav class="expandable
```

Comment:
Yep, lets do this separately, because it's going to require a whole bunch of larger changes. I've added a task for it, if I have time I'll start it this sprint.